e2e/suites/management/add_rooms_test.go

```diff
@@ -0,0 +1,249 @@
+// MIT License
+//
+// Copyright (c) 2021 TFG Co
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package management
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/go-redis/redis"
+	roomStorageRedis "github.com/topfreegames/maestro/internal/adapters/room_storage/redis"
+	"github.com/topfreegames/maestro/internal/core/entities/game_room"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/topfreegames/maestro/e2e/framework/maestro"
+
+	"github.com/topfreegames/maestro/e2e/framework"
+	maestrov1 "github.com/topfreegames/maestro/pkg/api/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func TestAddRooms(t *testing.T) {
+	framework.WithClients(t, func(apiClient *framework.APIClient, kubeclient kubernetes.Interface, redisClient *redis.Client, maestro *maestro.MaestroInstance) {
+
+		roomsStorage := roomStorageRedis.NewRedisStateStorage(redisClient)
+
+		t.Run("when created rooms does not reply its state back then it finishes the operation successfully", func(t *testing.T) {
```

Comment:
This test description is not so clear to me, reading the test code, I believe that you're just testing the happy path, that's right?

Reply:
Yes, even if the game rooms does no reply its ready state back or some error occur when each of the rooms, we consider the operation finished with success. The only sad path that I can identify is when the scheduler is not found when adding rooms, I'm going to add this test case!.

Reply:
c350ce2